cmake-proxies/cmake-modules/Package.cmake

```diff
@@ -2,7 +2,7 @@ set(CPACK_PACKAGE_VERSION_MAJOR "${AUDACITY_VERSION}") # X
 set(CPACK_PACKAGE_VERSION_MINOR "${AUDACITY_RELEASE}") # Y
 set(CPACK_PACKAGE_VERSION_PATCH "${AUDACITY_REVISION}") # Z
 set(CPACK_PACKAGE_VENDOR "Saucedacity")
-set(CPACK_PACKAGE_HOMEPAGE_URL "https://saucedacity.github.io")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://tenacity.github.io")
```

Comment:
Careful.

Reply:
Wait, `tenacity` isn't ours and the website is different.

Reply:
Incorrect; this was an automation error. Will be correct to `https://tenacityaudio.org`.